views/layouts/layout-example.njk

```diff
@@ -1,31 +1,50 @@
-<!doctype html>
-<html lang="en">
-  <head>
-    <meta charset="utf-8">
-    <meta name="robots" content="noindex, nofollow">
-    <meta name="viewport" content="width=device-width, initial-scale=1">
-    <title>{{ title }} – Example – GOV.UK Design System</title>
-    <link href="{{ getFingerprint('stylesheets/main.css') }}" rel="stylesheet" media="all" />
-    {#- Include any additional stylesheets specified in the example frontmatter #}
-    {% for stylesheet in stylesheets %}
+{% extends "govuk/template.njk" %}
+
+{% set htmlClasses = 'no-js' %}
+{% set assetUrl = 'https://design-system.service.gov.uk/assets' %}
+
+{% block pageTitle %}{{ title }} – Example - GOV.UK Design System{% endblock %}
+
+{% block head %}
+  <meta name="robots" content="noindex, nofollow">
+  <meta name="og:title" content="{{title}}">
+  <meta name="description" content="{{description}}">
+  <meta name="og:description" content="{{description}}">
+  <link rel="canonical" href="{{ canonical }}" />
+
+  <link href="{{ getFingerprint('stylesheets/main.css') }}" rel="stylesheet" media="all" />
+  {#- Include any additional stylesheets specified in the example frontmatter #}
+  {% for stylesheet in stylesheets %}
     <link href="{{ getFingerprint(stylesheet) }}" rel="stylesheet" media="all" />
-    {%- endfor %}
-
-    <!--[if lt IE 9]>
-      <script src="{{ getFingerprint('javascripts/head-ie8.js') }}"></script>
-    <![endif]-->
-    <script src="/javascripts/vendor/modernizr.js"></script>
-  </head>
-  <body class="govuk-template__body app-example-page">
-    <script>document.body.className = ((document.body.className) ? document.body.className + ' js-enabled' : 'js-enabled');</script>
-    <!-- Disable browser validation for any examples that include form elements -->
-    <form action="/form-handler" method="post" novalidate>
-      {% block body %}
-        {{ contents | safe }}
-      {% endblock %}
-    </form>
-    <script src="/javascripts/vendor/iframeResizer.contentWindow.js"></script>
-    <script src="{{ getFingerprint('javascripts/govuk-frontend.js') }}"></script>
-    <script src="{{ getFingerprint('javascripts/example.js') }}"></script>
-  </body>
-</html>
+  {%- endfor %}
+
+  <!--[if lt IE 9]>
+    <script src="{{ getFingerprint('javascripts/head-ie8.js') }}"></script>
+  <![endif]-->
+  <script src="/javascripts/vendor/modernizr.js"></script>
+{% endblock %}
+
+{# Example pages shouldn't have a skip link or header, so blank the one provided by the template #}
+{% block skipLink %}{% endblock %}
+{% block header %}{% endblock %}
+
+{% set htmlClasses = "app-example-page__wrapper" %}
```

Comment:
We're setting `htmlClasses` twice – also on line 3. Is that intentional?

Reply:
@36degrees Thanks, I've fixed that. I removed the line to add `no-js` to `htmlClasses` as it looks like it's only used by the design system navigation which this page won't have anyway.